tests/workflow_contracts/namespace_runners_test.py

```
@@ -0,0 +1,93 @@
+
+
+def test_ci_uses_github_hosted_linux_for_the_whitaker_toolchain() -> None:
+    """Keep CI on a runner compatible with Whitaker's prebuilt cargo-dylint."""
+    workflow_path = _REPO_ROOT / ".github" / "workflows" / "ci.yml"
+    workflow = yaml.safe_load(workflow_path.read_text(encoding="utf-8"))
+    assert workflow["jobs"]["build-test"]["runs-on"] == "ubuntu-latest"
+
+
+@pytest.mark.parametrize(
+    ("delay_minutes", "expected_seconds"),
+    (("1", "60"), ("0", "0"), ("08", "480")),
```

Ruff (0.16.3)

[warning] Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)